packages/react/src/components/Listbox/ListboxGroup.tsx

```diff
@@ -0,0 +1,37 @@
+import { ContentNode } from '../../types';
+import React, { forwardRef } from 'react';
+import { useId } from 'react-id-generator';
+
+interface ListboxGroupProps extends React.HTMLAttributes<HTMLElement> {
+  as?: React.ElementType | string;
+  groupLabelProps?: React.HTMLAttributes<HTMLLIElement>;
+  label: ContentNode;
+}
+
+const ListboxGroup = forwardRef<HTMLElement, ListboxGroupProps>(
+  (
+    {
+      as: Component = 'ul',
```

Comment:
I wonder what if a user changes `as` into a `div`? The we will have `li` under `div`?

Reply:
Correct. We don't have control over if the author changes the element to something that might not be valid markup, we just provide the hook to do so.